hmpps-sqs-spring-boot-autoconfigure/src/main/kotlin/uk/gov/justice/hmpps/sqs/HmppsQueueFactory.kt

```diff
@@ -138,9 +152,14 @@ class HmppsQueueFactory(
     dlqName: String,
     maxReceiveCount: Int,
     visibilityTimeout: Int,
+    fifoQueue: String?,
+    fifoThroughputLimit: String?,
   ) = runBlocking {
+    val fifoAttributes = when {
+      fifoQueue == "true" -> mapOf(QueueAttributeName.FIFO_QUEUE to fifoQueue, QueueAttributeName.FIFO_THROUGHPUT_LIMIT to fifoThroughputLimit) else -> mapOf()
+    }
```

Comment:
FIFO_QUEUE must be either "true" or not supplied at all
 https://github.com/aws/aws-cdk/issues/8550